roles/addons/users_basic/readme.rst

```diff
@@ -9,9 +9,43 @@ This role provides a very basic users management, for simple clusters.
 Instructions
 ^^^^^^^^^^^^
 
-Copy addon/users_basic.yml into /etc/bluebanquise/inventory/group_vars/all/addons folder.
+Create file inventory/group_vars/all/addons/users_basic.yml with the following content:
 
-Then edit this file according to your needs, and play this role on all hosts with users.
+.. code-block:: yaml
+
+  users_basic:
+
+    users:
+      - name: johnnykeats
+        uid: 1078
+        gid: 1078
+        home: /home
+        shell: /bin/bash
+        comment: dream all the day # optional
+        password: $6$M3crarMVoUV3rALd$ZTre2CIyss7zOb4lkLoG23As9OAkYPw2BM88Y1F43n8CCyV5XWwAYEwBOrS8bcCBIMjIPdJG.ndOfzWyAVR4j0 # Only set the first time, when user does not exist, then ignored
```

Comment:
`update_password` defaults to `always` in the role.
```suggestion
        password: $6$M3crarMVoUV3rALd$ZTre2CIyss7zOb4lkLoG23As9OAkYPw2BM88Y1F43n8CCyV5XWwAYEwBOrS8bcCBIMjIPdJG.ndOfzWyAVR4j0
```

Reply:
Yes, removed comment.